DOCS/interface-changes.rst

```diff
@@ -99,6 +99,8 @@ Interface changes
     - deprecate `--demuxer-cue-codepage` in favor of `--metadata-codepage`
     - change the default of `metadata-codepage` to `auto`
     - add `playlist-next-playlist` and `playlist-prev-playlist` commands
+    - enabling memory reporting for javascript scripts is now done by exporting the
+      JS_MEM_REPORT environment variable
```

Comment:
This change makes sense regardless of what happens to the ta leak reporter. Though I'd "namespace" the env var to something like `MPV_JS_MEM_REPORT` just in case.

Reply:
Is there a reason to not just turn it an option?

Reply:
I don't think the TA subsystem interacts with mpv's option, but I could be wrong.

Reply:
We don't currently have any options specific to any scripting backend I believe (aside from loading them or not). I'm honestly fine with any of the various proposed changes: changing the environment variable name, making an option, or just enabling it by default all the time. I guess it uses more memory technically but does it really matter?

Reply:
Personally I don't think it matters how to enable it, as long as it's documented.